app/code/Magento/Customer/Controller/Address.php

```diff
@@ -89,12 +95,14 @@ public function __construct(
         \Magento\Framework\Reflection\DataObjectProcessor $dataProcessor,
         \Magento\Framework\Api\DataObjectHelper $dataObjectHelper,
         \Magento\Framework\Controller\Result\ForwardFactory $resultForwardFactory,
-        \Magento\Framework\View\Result\PageFactory $resultPageFactory
+        \Magento\Framework\View\Result\PageFactory $resultPageFactory,
+        ?\Magento\Customer\Api\CustomerRepositoryInterface $customerRepository
```

Comment:
I don't think this syntax (question mark) will work as intended. It will allow passing the argument of type `null`, but result in "Too few arguments" fatal error if no argument is passed. Please specify the default `null` value to make the new added dependency optional